src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -63,6 +64,15 @@
 public interface RqMultipart extends Request {
 
     /**
+     * Carriage return constant.
+     */
+    String CRLF = "\r\n";
+    /**
+     * Content type constant.
+     */
+    String CONTENT_TYPE = "Content-Type";
```

Comment:
why these too are visible to users of the interface? they should be private, inside classes that use them